### ISSUE TITLE
Save the value of "Specify poll interval" into the SQL Developer into the properties.

### ISSUE THREAD

**Issue by mcalicchio** [scristalli/SQL-Developer-4-keepalive#28]
Writing the value the user entered into the properties file would be helpful so he/she wouldn't need to keep reentering that value every time the app comes up.
Also, if we could just write that value in a property value, the app would only need to load it so no need to enter it every time.

**Comment by stepheneire** [comment]
Yes this would be very useful. It's super annoying having to enter it every time I start Sql Developer.
